packages/xstate-test/src/utils.ts

```diff
@@ -78,6 +78,9 @@ export function formatPathTestResult(
 export function getDescription<T, TContext>(state: AnyState): string {
   const contextString =
     state.context === undefined ? '' : `(${JSON.stringify(state.context)})`;
```

Comment:
Let's call this in a more generic way - this is not bound to jest nor to console. The whole function just produces some text description

Reply:
That makes sense. I switched to `transitionDescription` as this would be the description on the transition taken by the event. Happy to change it if you have a more appropriate name.